monkey/agent_plugins/exploiters/mssql/src/mssql_exploit_client.py

```diff
@@ -69,12 +70,18 @@ def exploit_host(
         """
         exploitation_message = ""
         exploitation_success = True
+        ports_to_try = set(options.target_ports)
+
+        if options.try_discovered_mssql_ports:
+            relevant_tcp_ports_on_host = get_open_tcp_ports(
+                host=host, services_to_filter={NetworkService.MSSQL, NetworkService.UNKNOWN}
+            )
+            ports_to_try.update(relevant_tcp_ports_on_host)
```

Comment:
Option name doesn't match with the behavior. This is also trying ports whose services are unknown.

Reply:
I also don't think not trying unknown service ports is a good idea. Maybe we should change the option name or add another one?

Reply:
Let's just add text to the configuration option mentioning that ports with unknown services will also be tried.